CHANGELOG.md

```diff
@@ -1,12 +1,22 @@
 # Braintree Android SDK Release Notes
 
-## unreleased
 * Breaking Changes
   * All Modules
     * Bump `minSdkVersion` to API 23
   * UnionPay
     * Remove `union-pay` module
       * UnionPay cards can now be processed as regular cards (through the `card` module) due to their partnership with Discover
+  * BraintreeDataCollector
+    * Remove `data-collector` module (use `paypal-data-collector`) 
```

Comment:
Ah, based on our Slack convo I thought we were going to ditch `paypal-data-collector` for `data-collector` for the module name